lib/offsite_payments/integrations/paytm.rb

```diff
@@ -207,6 +207,8 @@ def checksum_ok?
             @message = 'Return checksum not matching the data provided'
             false
           end
+        rescue
```

Comment:
is it possible to still log the error, in case we find a new error in the future?

Reply:
I'll see what I can do!

Reply:
I can't figure out how we'd log here. Also, do we care? We only want to know if the payload is ok, not what's missing from it.

Reply:
ok, no problemo, (only mentioned it cause `rescue` will swallow all errors in that block) it's not too important though